src/analyses/variable-sensitivity/variable_sensitivity_configuration.h

```diff
@@ -51,18 +52,15 @@ struct vsd_configt
 
   flow_sensitivityt flow_sensitivity;
 
+  size_t maximum_array_index = std::numeric_limits<size_t>::max();
```

Comment:
I presume this is pending future changes to option handling?

Reply:
Yes, that's right.